### ISSUE TITLE
Clicking edit-saved search should show edit-saved-search-modal

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#2207]
### Expected behaviour
When I click the edit-button for saved-searches in the filter-dropdown, the edit-saved-search-modal should be visible.
![screen shot 2017-10-23 at 16 34 24](https://user-images.githubusercontent.com/8624777/31895013-5de35aee-b810-11e7-85a2-766912c022db.png)

### Actual behaviour
Only delete-saved-searches should be visible. 

##### Where
- [ ] [Local setup with Vagrant ](https://www.ushahidi.com/support/install-ushahidi#installing-for-development)
- [ ] [Local setup from platform-release ](https://www.ushahidi.com/support/install-ushahidi#installing-the-latest-release)
- [ ] Ushahidi.io / SaaS solution 
- [ ] Ushahidi's QA environment
- [ ] Other (explain):

**Comment by jshorland** [comment]
I'm putting this as p2 because you can edit saved searches from the map context bar. As long as there's a path to get there and @aoduor knows how to do it, there are more important issues to deal with.

**Comment by jshorland** [comment]
Confirmed. Still an issue.

**Comment by crcommons** [comment]
Currently sitting in uirouter-migration-init branch. Once that is merged into Develop, this is ready for QA.
